ui/app/AppLayouts/Wallet/adaptors/TokenSelectorViewAdaptor.qml

```diff
@@ -23,6 +23,7 @@ QObject {
       - balances: submodel -> [ chainId:int, account:string, balance:BigIntString, iconUrl:string ]
 
       Computed values:
+      - currentBalance: double (amount of tokens)
```

Comment:
nit: naming could be more explanatory. The ones below are are also "current", right? The difference is that some are in `fiat` units and some are in `crypto` units)